website/docs/guides/delivery.mdx

```diff
@@ -0,0 +1,473 @@
+---
+title: Progressive Delivery using Flagger
+hide_title: true
+sidebar_position: 6
+---
+
+import TierLabel from "../_components/TierLabel";
+
+# Progressive Delivery using Flagger <TierLabel tiers="enterprise" />
+
+Weave GitOps Enterprise includes the ability to view [Flagger](https://docs.flagger.app/) [canary](https://docs.flagger.app/usage/how-it-works#canary-resource) objects information in the UI through the Applications > Delivery view.
```

Comment:
The two hyperlinks next to each other may make this a little confusing, and we could probably sell this a little more.

 Suggested rework:
"[Flagger](https://docs.flagger.app/) is a progressive delivery operator for Kubernetes. It is designed to reduce risks when introducing new software versions and to improve time to delivery through automating production releases. Weave GitOps Enterprise's UI allows you to view the state of these progressive delivery rollouts, and how they are configured using Flagger's [canary](https://docs.flagger.app/usage/how-it-works#canary-resource) object, through the Applications > Delivery view."

Reply:
Updated 👍